packages/flex-plugin-scripts/src/config/devServer/webpackDevServer.ts

```diff
@@ -0,0 +1,73 @@
+import WebpackDevServer, { Configuration } from 'webpack-dev-server';
```

Comment:
Moved this out because it can be standalone and makes the `start` script cleaner